src/hooks/useCreateEditCoupon.ts

```diff
@@ -29,6 +29,7 @@ gql`
     name
     amountCurrency
     code
+    reusable
```

Comment:
Is that change needed ? Seems `reusable` is not used in here

Reply:
Hmm, If I remember correctly it was needed (this is edit coupon fragment) since reusable checkbox is also editable...

Reply:
ok probably an issue with the fragment usage in here, that was already done before you jumped into this file.

Will resolve, as a refactor can come later